Extensions/popupLyrics.js

```diff
@@ -1,4 +1,4 @@
-// @ts-check
+// @ts-nocheck
```

Comment:
```suggestion
// @ts-check
```

I'm not sure why you removed it, but you should get minimal errors with the check. Some are miscategorized, we are aware, but what you get from the `check` is more than what you'll get by just having it disabled.

Reply:
I would love to but it throws error that I can't fix since it's about types. This file isn't typescript one so I can't add types.

Reply:
Having typescript check in JavaScript file isn't ideal and shouldn't be done at all because of errors. Typescript is for types, not for no having them and only using it for example "this return statement isn't going to work" etc etc. If you really need it I can simply add typescript file that will generate JavaScript dist